### PR TITLE
Chicken base fix

### PR DESCRIPTION
**Pull request by Gingerbeard5773**
Adds Chicken Variant support to the new base types Stronghold & Convent, which previously did not have their own chicken variants- which caused bugs and issues regarding capture.

### CODE REVIEW

File: TFlippy_TerritoryControl_Characters_Dev/Entities/NPC/CombatChickens/ChickenCitadel/ChickenCitadel.as [hunk 1]
```diff
@@ -42,10 +42,17 @@ void onTick(CBlob@ this)
 				CBlob@[] chickens;
 				getBlobsByTag("combat chicken", @chickens);
 				
-				if (chickens.length < 12)
-				{
-					CBlob@ blob = server_CreateBlob((XORRandom(100) < 75 ? "heavychicken" : "soldierchicken"), -1, this.getPosition() + Vec2f(16 - XORRandom(32), 0));
-				}
+				if (chickens.length < 8)
+                {
```
Comment (Vam-Jam) [inline TFlippy_TerritoryControl_Characters_Dev/Entities/NPC/CombatChickens/ChickenCitadel/ChickenCitadel.as]: fix indent

File: TFlippy_TerritoryControl_Characters_Dev/Entities/NPC/CombatChickens/ChickenFortress/ChickenFortress.as [hunk 2]
```diff
@@ -41,9 +41,16 @@ void onTick(CBlob@ this)
 				getBlobsByTag("combat chicken", @chickens);
 				
 				if (chickens.length < 8)
-				{
-					CBlob@ blob = server_CreateBlob((XORRandom(100) < 75 ? "soldierchicken" : "scoutchicken"), -1, this.getPosition() + Vec2f(16 - XORRandom(32), 0));
-				}
+                {
```
Comment (Vam-Jam) [inline TFlippy_TerritoryControl_Characters_Dev/Entities/NPC/CombatChickens/ChickenFortress/ChickenFortress.as]: fix indent

Comment (Gingerbeard5773) [reply TFlippy_TerritoryControl_Characters_Dev/Entities/NPC/CombatChickens/ChickenFortress/ChickenFortress.as]: Fixed

File: TFlippy_TerritoryControl_Characters_Dev/Entities/NPC/CombatChickens/ChickenCitadel/ChickenCitadel.as (hunk 1 above)
Comment (Gingerbeard5773) [reply TFlippy_TerritoryControl_Characters_Dev/Entities/NPC/CombatChickens/ChickenCitadel/ChickenCitadel.as]: Fixed

**Pull request merged**